eea/reports/browser/report.pt

```diff
@@ -531,7 +531,7 @@
             </li>
           </tal:link>
 
-          <li tal:condition="python:item.portal_type=='Folder'" tal:attributes="
+          <li tal:condition="python:item.portal_type == 'Folder'" tal:attributes="
```

Comment:
@alecghica  I propose that any Folderish item should be expanded in the toc and you can do so with item.isPrincipiaFolderish which if the content is folderish will return 1 and 0 otherwise making it useful for the condition.